server/workspace-service/src/service.ts

```diff
@@ -188,8 +189,9 @@ export class WorkspaceWorker {
         await this.doSleep(ctx, opt)
       } else {
         void this.exec(async () => {
+          const operationJobUuid = randomUUID()
           await ctx
-            .with('workspaceOperation', { mode: workspace.mode }, (ctx) =>
+            .with('workspaceOperation', { operationJobUuid, mode: workspace.mode }, (ctx) =>
```

Comment:
it is not logging, it will use huge set of memory, please use regular logging instead with context